build-ap-microxrceddsgen-deb.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <version>"
+  exit 1
+fi
+
```

Comment:
```suggestion

set -euf -o pipefail

```

Catch errors earlier with this protection in shell.